api/src/test/java/org/pmiops/workbench/api/NotebooksControllerTest.java

```diff
@@ -118,522 +104,188 @@ class MockNotebook {
-    String fromNotebookName = "origin";
-
-    DbWorkspace toWorkspace = createWorkspace("toWorkspaceNs", "toworkspace");
-    String newNotebookName = "new";
-    String expectedNotebookName = newNotebookName + NotebooksService.NOTEBOOK_EXTENSION;
-
-    stubGetWorkspace(fromWorkspace, WorkspaceAccessLevel.OWNER);
-    stubGetWorkspace(toWorkspace, WorkspaceAccessLevel.OWNER);
-
-    notebooksController.copyNotebook(
-        fromWorkspace.getWorkspaceNamespace(),
-        fromWorkspace.getFirecloudName(),
-        fromNotebookName,
-        new CopyRequest()
-            .toWorkspaceName(toWorkspace.getFirecloudName())
-            .toWorkspaceNamespace(toWorkspace.getWorkspaceNamespace())
-            .newName(newNotebookName));
-
-    verify(mockCloudStorageClient)
-        .copyBlob(
-            BlobId.of(
-                TestMockFactory.WORKSPACE_BUCKET_NAME,
-                "notebooks/" + NotebooksService.withNotebookExtension(fromNotebookName)),
-            BlobId.of(TestMockFactory.WORKSPACE_BUCKET_NAME, "notebooks/" + expectedNotebookName));
-
-    verify(mockUserRecentResourceService)
-        .updateNotebookEntry(
-            toWorkspace.getWorkspaceId(),
-            currentUser.getUserId(),
-            BUCKET_URI + "notebooks/" + expectedNotebookName);
-  }
+    Throwable exception =
+        assertThrows(
+            BadRequestException.class,
+            () ->
+                notebooksController.cloneNotebook(
+                    FROM_WORKSPACE_NAMESPACE, FROM_WORKSPACE_NAME, FROM_NOTEBOOK_NAME));
 
-  @Test
-  public void copyNotebook_onlyAppendsSuffixIfNeeded() {
-    DbWorkspace fromWorkspace = createWorkspace();
-    String fromNotebookName = "origin";
-
-    DbWorkspace toWorkspace = createWorkspace("toWorkspaceNs", "toworkspace");
-    String newNotebookName = NotebooksService.withNotebookExtension("new");
-
-    stubGetWorkspace(fromWorkspace, WorkspaceAccessLevel.OWNER);
-    stubGetWorkspace(toWorkspace, WorkspaceAccessLevel.OWNER);
-
-    notebooksController.copyNotebook(
-        fromWorkspace.getWorkspaceNamespace(),
-        fromWorkspace.getFirecloudName(),
-        fromNotebookName,
-        new CopyRequest()
-            .toWorkspaceName(toWorkspace.getFirecloudName())
-            .toWorkspaceNamespace(toWorkspace.getWorkspaceNamespace())
-            .newName(newNotebookName));
-
-    verify(mockCloudStorageClient)
-        .copyBlob(
-            BlobId.of(
-                TestMockFactory.WORKSPACE_BUCKET_NAME,
-                "notebooks/" + NotebooksService.withNotebookExtension(fromNotebookName)),
-            BlobId.of(TestMockFactory.WORKSPACE_BUCKET_NAME, "notebooks/" + newNotebookName));
+    assertThat(exception.getMessage()).isEqualTo("File already exists at copy destination");
+    verify(mockNotebookService)
+        .cloneNotebook(FROM_WORKSPACE_NAMESPACE, FROM_WORKSPACE_NAME, FROM_NOTEBOOK_NAME);
   }
 
   @Test
-  public void copyNotebook_onlyHasReadPermissionsToDestination() {
```

Comment:
noting that these 2 tests are removed because we are already testing that the `to` and `from` access levels are tested - these are eequivalent.